.rubocop.yml

```diff
@@ -1,63 +1,72 @@
+
+# List of cops can be found here:
+# https://github.com/bbatsov/rubocop/blob/master/config/default.yml
+# https://github.com/bbatsov/rubocop/blob/master/config/disabled.yml
+# https://github.com/bbatsov/rubocop/blob/master/config/enabled.yml
+
 inherit_from: .rubocop_todo.yml
 
 AllCops:
   DisplayCopNames: true
-  Include:
-    - '**/Rakefile'
-    - '**/config.ru'
+  DisplayStyleGuide: true
   Exclude:
     - 'db/**/*'
-    - 'script/**/*'
-    - 'spec/internal/**/*'
-    - 'spec/test_app_templates/**/*'
-    - 'vendor/**/*'
-
-Lint/ImplicitStringConcatenation:
-  Exclude:
-    - 'lib/generators/sufia/**/*'
+    - 'tmp/**/*'
+  ExtraDetails: true
+  TargetRubyVersion: 2.2
 
-Metrics/LineLength:
-  Max: 400
+# Alot of these metrics are quite restrictive. We may want to disable a few of these.
+# For now let's keep them and increase them by an excessive amount
 
+# A calculated magnitude based on number of assignments,
+# branches, and conditions.
 Metrics/AbcSize:
-  Max: 30
-  Exclude:
-    - 'app/controllers/concerns/sufia/admin/depositor_stats.rb'
-    - 'app/controllers/concerns/sufia/my_controller_behavior.rb'
-    - 'app/services/sufia/user_stat_importer.rb'
-    - 'lib/sufia/arkivo/metadata_munger.rb'
+  Max: 45 # default 15
 
+Metrics/ClassLength:
+  Max: 500 # default 100
+
+# A complexity metric that is strongly correlated to the number
+# of test cases needed to validate a method.
+Metrics/CyclomaticComplexity:
+  Max: 17 # default 6
+
+Metrics/LineLength:
+  Enabled: true
+  Max: 120 # default is 80
```

Comment:
I'm a big fan of 80 myself (the human visual system doesn't scan long lines of text well, which is why historically print media breaks articles into multiple columns).

My bias aside, if the goal for this project is to push more code into the community, it might be worth checking what other related projects use (hydra, sufia, blacklight). This advice would apply to other settings in rubocop too.